istio/istio-install/base/istio-galley-configuration-validatingwebhookconfiguration.yaml

```diff
@@ -0,0 +1,116 @@
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: ValidatingWebhookConfiguration
+metadata:
+  name: istio-galley
+  labels:
+    app: galley
+    chart: galley
+    heritage: Tiller
```

Comment:
@krishnadurai 
This was removed by @jbrette in PR #378 but he closed it because we reverted.
Could you remove `heritage: Tiller`

Reply:
I'll remove these.